omnibus/cookbooks/firezone/attributes/default.rb

```diff
@@ -135,6 +135,10 @@
 #   }
 # }
 
+# ## Proxy
+default['firezone']['trusted_proxy'] = []
```

Comment:
suggestion to call this `trusted_proxies` since it's an array

Reply:
Also could you add a comment with details explaining what this is? Also isn't '127.0.0.1' required by default? E.g.

```ruby
# An array of trusted reverse proxies that Firezone should accept downstream requests from. By default, this is 
# only the bundled nginx proxy at 127.0.0.1, but you may want to use other proxies such as Amazon ELB, 
# CloudFlare DDoS protection, etc.
default['firezone']['trusted_proxies] = ['127.0.0.1']
```

Reply:
`127.0.0.1` is always trusted. Will add a comment explaining more about [this](https://hexdocs.pm/remote_ip/RemoteIp.Options.html#module-proxies) and I will add a [`clients`](https://hexdocs.pm/remote_ip/RemoteIp.Options.html#module-clients) option also.

Reply:
Fixed [here](https://github.com/firezone/firezone/pull/795/commits/6030cf1cd0f5a93d749e18a428ba1bc7f916df5e)